### DISCUSSION TITLE
Новые расы для донатеров (by Дмитрий Пивная Бочка)

### DISCUSSION THREAD

**Discussion by kaworu177** [space-sunrise/space-station-14#603] [Ideas]
Сразу говорю, этот бред писал не я, а другой чел из дискорда, я просто перенёс по его просьбе.
![image](https://github.com/user-attachments/assets/67ba1b95-e0be-442f-8e93-20a58e9b21af)

1.Врин (множ. Врины) — раса пчелоподобных инсектоидов, не нуждающихся в дыхании и невосприимчивы к низкому 
давлению, могут строить благодаря восковым железам, пчёлы не атакуют Вринов. Имеют сотовую связь улья благодаря своим усикам (:у)  и чувствуют когда кто то из их вида умер. Из минусов можно отметить  неспособность носить скафандры, так как они перегружают хрупкие тела Вринов и делают их неподвижными, слабость в рукопашном бою и медлительность, невозможность клонирования, также высокие или низкие температуры вредны для Вринов
Рост: 145-170 см
Продолжительность жизни: 45-80 лет 
![WrynM](https://github.com/user-attachments/assets/ac3ba806-d8ae-463c-9237-ad1285c160b1)

2.Драск (множ. Драски) — раса гуманоидных хладнокровных существ. Способны усваивать мыло, сердце расположено в голове, вместо крови течёт чистый криоксадон. Из плюсов можно отметить: способность к регенерации при низких температурах как внешней так и внутри Драска, имеют полный иммунитет к повреждениям от низких температур, урон голых рук увеличен. Но есть и минусы: вспышки наносят увеличенный урон глазам, не могут восстанавливать кровь пакетами крови, физраствором и железом, получают повышенный урон от высоких температур, плохо переносят сладкую пищу. 
Рост: 235 см
Продолжительность жизни: Биологически бессмертны 
Дыхательный газ: Кислород 
![DraskM](https://github.com/user-attachments/assets/2bf8cc48-85ca-4ee6-9e52-61c723b483d5)

3.Кидан (множ. Киданы) — раса муравьиноподобных гуманоидов обладающих экзоскелетом. У Киданов мозг находится в торсе, они обладают биолюминесценцией (способностью живых существ светиться), может потреблять нимф дион, обладают эхолокатором руды на эмоции трещания и щёлкания челюстями, что делает их превосходными утилизаторами, в рукопашном бою атакуют когтями.
Рост: 190 см
Продолжительность жизни: 55 лет
Дыхательный газ: кислород 
![KidanM](https://github.com/user-attachments/assets/7a0ccd38-65fa-4a4c-83b9-3099c8eced4b)

4.Скрелл (множ. Скреллы) — раса гуманоидных амфибий. Имеют небольшое сопротивление токсинам, орган головного кармана, вмещающий в себя один маленький предмет. Вода лечит скреллов, а алкоголь смертельно опасен для них. 
Рост: 180-185 см
Продолжительность жизни: 80-400 лет
Дыхательный газ: Кислород
![SkrellM](https://github.com/user-attachments/assets/5fbd74ca-6dff-4663-bffe-69b88c353159)

**Comment by kaworu177** [comment]
Я этот бред даже не читал лол.

**Comment by andreySS14** [comment]
Врин - тот же ксенос но с иными приколами, кидан - онли утили, скрелл - сойдёт, драск - сойдёт
